app/models/short_url_request.rb

```diff
@@ -25,6 +25,16 @@ class ShortUrlRequest
 
   scope :pending, -> { where(state: "pending") }
 
+  attr_accessor :confirmed
+
+  def duplicates
+    @duplicates ||= Redirect.or({from_path: from_path}, {to_path: to_path})
```

Comment:
I wonder if we want to be more strict about what our definition of a "duplicate" request is. We have a story in progress which prevents "full duplicates"  - as in, requests that match the from path and to path for an existing live redirect. Is there therefore a case for naming this method to something else to make this distinction clear, like `redirects_with_matching_paths`?